config/locales/pt.yml

```diff
@@ -0,0 +1,10 @@
+pt:
+  activerecord:
+    attributes:
+      user:
+        name: "Nome"
+        submit: "Cenas"
+    errors:
+      messages:
+        blank: "Não pode estar em branco"
+        invalid: "É inválido"
```

Comment:
We're missing "has already been taken". It should be something como "Já está inscrito"

Reply:
Yeah you are right. Maybe the translation should be "Utilizador já existe"